src/50_Samples_%26_Templates/Poll.html

```diff
@@ -90,7 +90,7 @@ <h4 id="header1">Flightless bird poll</h4>
 accessing a specific origin, for example the original page or the cached version of it, this means that the same user will generate two different
 values when accessing different origins. That's why we use a custom cookie to track cross origin access to the poll page.
 The `CLIENT_ID` variable can be passed via an amp-form by declaring an hidden input
-value with `default-value="CLIENT_ID(POLL_USER_ID)`. We are using the `CLIENT_ID` as the value of the `POLL_USER_ID` cookie, but in our implementation the value of this cookie doesn't really matter, in fact it could also be a random number.
+value with `default-value="CLIENT_ID(POLL_USER_ID)`. In our implementation, the backend sets `POLL_USER_ID` to the value of `CLIENT_ID`, but the value of this cookie doesn't really matter.
```

Comment:
s/doesn't really matter/is not relevant and you could re-use your own session id./